src/bp-templates/bp-nouveau/includes/search/functions.php

```diff
@@ -57,12 +57,22 @@ function bp_nouveau_search_enqueue_scripts() {
 		'per_page'              => $per_page,
 		'autocomplete_selector' => "form[role='search'], form.search-form, form.searchform, form#adminbarsearch, .bp-search-form>#search-form",
 		'form_selector'         => '',
+		'forums_autocomplete'   => false,
 	);
 
 	if ( isset( $_GET["s"] ) ) {
 		$data["search_term"] = $_GET["s"];
 	}
 
+	if ( bp_is_active( 'forums' ) ) {
+		$data['forums_autocomplete'] = (
+			bp_is_search_post_type_enable( 'forum' ) ||
```

Comment:
```suggestion
			bp_is_search_post_type_enable( bbp_get_forum_post_type() ) ||
```
@janakkaneriya007 Use the function to get the dynamic value and it works always if someone change the slug.